docs/data/joy/getting-started/templates/sign-in-side/App.tsx

```diff
@@ -0,0 +1,290 @@
+                    '--joy-shadowChannel': (theme) =>
+                      theme.vars.palette.primary.mainChannel,
+                  }}
+                />
+              }
+            >
+              Logo
+            </Typography>
+            <ColorSchemeToggle />
+          </Box>
+          <Box
+            component="main"
+            sx={{
+              my: 'auto',
+              py: 2,
+              pb: 5,
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 2,
+              width: 400,
+              maxWidth: '100%',
+              mx: 'auto',
+              borderRadius: 'sm',
+              '& form': {
+                display: 'flex',
+                flexDirection: 'column',
+                gap: 2,
+              },
+            }}
+          >
+            <div>
+              <Typography component="h2" fontSize="xl2" fontWeight="lg">
+                Welcome back
+              </Typography>
+              <Typography level="body2" sx={{ my: 1, mb: 3 }}>
+                Let&apos;s get started! Please enter your details.
+              </Typography>
+            </div>
+            <form
+              onSubmit={(event: React.FormEvent<SignInFormElement>) => {
+                event.preventDefault();
+                const formElements = event.currentTarget.elements;
+                const data = {
+                  email: formElements.email.value,
+                  password: formElements.password.value,
+                  persistent: formElements.persistent.checked,
+                };
+                alert(JSON.stringify(data, null, 2));
+              }}
+            >
+              <FormControl required>
+                <FormLabel slotProps={{ asterisk: { sx: { display: 'none' } } }}>
+                  Email
+                </FormLabel>
+                <Input placeholder="Enter your email" type="email" name="email" />
+              </FormControl>
+              <FormControl required>
+                <FormLabel slotProps={{ asterisk: { sx: { display: 'none' } } }}>
+                  Password
+                </FormLabel>
+                <Input placeholder="•••••••" type="password" name="password" />
+              </FormControl>
+              <Box
+                sx={{
+                  display: 'flex',
+                  justifyContent: 'space-between',
+                  alignItems: 'center',
+                }}
+              >
+                <Checkbox size="sm" label="Remember for 30 days" name="persistent" />
+                <Link fontSize="sm" href="#replace-with-a-link" fontWeight="lg">
+                  Forgot password
+                </Link>
+              </Box>
+              <Button type="submit" sx={{ width: '100%' }}>
+                Sign in
+              </Button>
+            </form>
+            <Button
+              variant="outlined"
+              color="neutral"
+              sx={{ width: '100%' }}
```

Comment:
Having a `fullWidth` prop would help with this :)

Reply:
Done. Honestly, I would like to remove this prop in the future 😅. It looks better to solely rely on `sx` prop for one-line customization like this and to remove the chance to have `support fullWidth prop in other component` issue.

Reply:
Agree, it's a separate discussion, it was just a thought while reviewing the file. My thought was, if you need it for the first real life example, it may mean many other people would need it do. I don't think we need to support it everywhere, but forms often do need it, so components used in forms are great candidates for the prop.